app/assets/javascripts/initializers/initializeDrawerSliders.js

```diff
@@ -1,57 +1,72 @@
+/* global swipeState InstantClick initializeSwipeGestures slideSidebar */
```

Comment:
This tells eslint that these functions are defined on the window in some other file.